lib/typescript/components/inputs/SearchField/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React, {createRef, useCallback} from 'react';
+
+import {ReactComponent as CloseIcon} from 'assets/images/icons/close.svg';
+import {ReactComponent as SearchIcon} from 'assets/images/icons/search.svg';
+import styles from './style.module.scss';
+
+type Props = {
+  id?: string;
+  placeholder?: string;
+  value: string;
+  setValue: (value: string) => void;
+  resetClicked?: () => void;
+  autoFocus?: boolean;
+  dataCy?: string;
+};
+
+export const SearchField = ({id, placeholder, value, setValue, resetClicked, autoFocus, dataCy}: Props) => {
+  const inputRef = createRef<HTMLInputElement>();
+  const resetButton = useCallback(() => {
```

Comment:
I don't think `useCallback` is necessary here. If it is, then `resetClicked` is missing from the deps list.